src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/IO/Base/I_IOErrorProcessor.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace Acuminator.Utilities.DiagnosticSuppression.IO
+{
+	/// <summary>
+	/// Interface for i/o error processing.
+	/// </summary>
+	public interface I_IOErrorProcessor
```

Comment:
inconsistent naming

Reply:
Hm and how it should be called? I just don't like the `IIOErrorProcessor` because of double I

Reply:
but what's wrong with double I? `IInternable`, `IInvoice`, `IIf` are classical examples

Reply:
Maybe I do not like it because of four vowels in a row. But ok, I'll fix it

Reply:
done